myriad-scheduler/src/main/java/org/apache/myriad/Main.java

```diff
@@ -259,7 +250,6 @@ private void startNMInstances(Injector injector) {
   private void initServiceConfigurations(MyriadConfiguration cfg, Injector injector) {
     LOGGER.info("Initializing initServiceConfigurations");
     ServiceProfileManager profileManager = injector.getInstance(ServiceProfileManager.class);
-    TaskConstraintsManager taskConstraintsManager = injector.getInstance(TaskConstraintsManager.class);
```

Comment:
The TaskConstraintManager did little more than check the number of ports, the cpu and mem were handled by ServiceResourceProfile and ExtendedResource.  I added a getPorts method which returns the needed ports analogous to how memory and cpu are set.